docs/OpenCLCOnVulkan.md

```diff
@@ -823,6 +823,13 @@ describe:
 
 All supported.
 
+Reading 3D images with a unormalised sampler is not allowed in SPIR-V. Thus
+OpenCL source code performing this operation is emulated by normalising the
+coordinate before reading the image using a normalised sampler.
+
+The normalisation of the coordinate can introduce an accuracy issue as the
+floatint point division used is not correctly rounded.
```

Comment:
Nits: switch to normalize for consistency with the rest of the codebase. Also, unormalized 3D images are not allowed in Vulkan (instead of SPIR-V).